apis/python/src/tiledbsoma/io.py

```diff
@@ -221,10 +228,8 @@ def from_anndata(
     #        experiment.set(experiment.uns)
 
     logging.log_io(
-        f"Wrote {experiment._nested_name}",
-        util.format_elapsed(
-            s, f"{experiment._indent}FINISH WRITING {experiment._nested_name}"
-        ),
+        "Wrote",
```

Comment:
what happens is:

```
Already exists; replacing: ./tiledbsoma-data/pbmc-small
Wrote
Wrote
Wrote
Wrote
Wrote
Wrote
Wrote
Wrote
Wrote
Wrote
Wrote ./tiledbsoma-data/pbmc-small
```

Reply:
This was removed because it was logging the (now non-existent) `.name` property`.  I will replace it with the Experiment `uri`, as that is the only analog available in this route.